blog/2024/09-10-release-roadmap-10.10.0/index.mdx

```diff
@@ -0,0 +1,81 @@
+- Monday, October 7th, 2024 (unstable `20241007`): Final planned unstable, second release candidate. **The release should basically be ready to go**, and any remaining bugfix PRs **should be merged before this unstable**.
+- Saturday, October 12th, 2024: Release of 10.10.0!
+
+### Possible Timeline Changes
+
+Since this is a much smaller and quicker release than 10.8.0 or 10.9.0 were, hopefully this is enough time to get everything sorted and ready to go. But if we find it's **not**, we reserve the right to add an extra week in one or both of the following places:
+
+- After the Feature freeze, if we find that there are any outstanding **major** bugfixes that require additional time to merge before the API freeze.
+- After the API lock, if we find that there are any outstanding **non-API-breaking** bugfixes that require additional time to merge before the final unstable.
+
+If neither of those happen, the release will proceed as planned; otherwise, an extra step will be added into the timeline after the given points that matches the previous stage. This might push the final unstable back to either October 14th or October 21st, and the final release to either October 19th or October 26th. If this is relevant to your interests please keep an eye on our [Announcements Matrix channel](https://matrix.to/#/#jellyfin-announce:matrix.org) for announcements.
+
+### Information for 3rd Party Clients
+
+After the API freeze, our API should ideally remain stable, though it's possible there might be minor changes up until the API lock step. Please feel free to begin testing compatibility and report any issues to us now, and keep an eye out on those dates for the final API specification.
+
+### Information for Contributors
+
+If you're contributing to Jellyfin and your feature PRs miss the deadline, **please don't fret**. We are committed to this shorter major release timeline going forward, so 10.11.0 is at most about 6 months away. Your feature should eventually make it in.
+
+If you wish to help by submitting a bugfix, please do so as soon as you can, as we'd like to get as many fixes in and tested before the API lock as possible, to give at least 2 weeks of final testing before the release. Ensure you clearly specify that it is a bugfix, and ensure you keep your changes to an absolute minimum needed to fix the bug. **Bugfix PRs will target the `master` branch until the final release**, at which point they will target the `release-10.10.z` branch for upcoming point releases.
+
+## How You Can Test
+
+Testing this release should be very easy, identical to 10.9.0. Since our pre-releases are "just" our unstable releases here, that means that following our normal "unstable" install process is all you need to do.
```

Comment:
```suggestion
Testing this release should be very easy, identical to 10.9.0. Since our pre-releases are "just" our unstable releases here, that means that following our normal "unstable" installation process is all you need to do.
```